code/game/objects/items/robot/robot_items.dm

```diff
@@ -1,35 +1,31 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
-
 /**********************************************************************
 						Cyborg Spec Items
 ***********************************************************************/
 //Might want to move this into several files later but for now it works here
 /obj/item/borg/stun
-	name = "Electrified Arm"
-	icon = 'icons/obj/decals.dmi'
-	icon_state = "shock"
-
-	attack(mob/M as mob, mob/living/silicon/robot/user as mob)
-		M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been attacked with [src.name] by [key_name(user)]</font>")
-		user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used the [src.name] to attack [key_name(M)]</font>")
-		if(M.ckey)
-			msg_admin_attack("[key_name_admin(user)] used the [src.name] to attack [key_name_admin(M)]")
-
-		if(!iscarbon(user))
-			M.LAssailant = null
-		else
-			M.LAssailant = user
-
-		user.cell.charge -= 30
-
-		M.Weaken(5)
-		if (M.stuttering < 5)
-			M.stuttering = 5
-		M.Stun(5)
-
-		for(var/mob/O in viewers(M, null))
-			if (O.client)
-				O.show_message("\red <B>[user] has prodded [M] with an electrically-charged arm!</B>", 1, "\red You hear someone fall", 2)
+	name = "electrified arm"
+	icon = 'icons/obj/items.dmi'
+	icon_state = "elecarm"
+	
+/obj/item/borg/stun/attack(mob/M, mob/living/silicon/robot/user)
+	user.cell.charge -= 30

```

Comment:
Um, shouldn't this have a max() to prevent it from going below zero? Or actually call the cell de-charge proc?